docs/release_notes.md

```diff
@@ -1,5 +1,160 @@
+- Remove `schema_validation_base_uri` from server_config.json (#7229).
+- Remove dependency on python 3 requests module (#7229).
+- Remove support for `AGE_KW` from DataObjTrim API (#7504).
+- Remove RBUDP (#7560).
+- Remove `msiSendMail` (#7563).
+- Remove `--link` option from `iput` and `irsync` (#7578).
+- Remove `BACKUP_RESC_NAME_KW` (#7594).
+- Remove `msiSetResource` (#7603).
+- Remove update_deprecated_database_columns.py script (#7834).
+- Remove server monitoring microservices  (#7904).
+- Remove `imeta qu` (#7960).
+- Remove interactive mode from `imeta` (#7962).
+- Remove DataObjLock and DataObjUnlock APIs (#7980).
+- Remove ability to disable Strict ACLs (#7954, #7985).
+- Remove `irodsctl` script (#7996).
+- Remove status function from IrodsController (#8008).
+- Remove SysVinit script (#8020).
+- Remove `schema_version` property from zone report (#8038).
+- Remove server_status.json.in (#8058).
+- Remove GeneralRowInsert API (#8068).
+- Remove GeneralRowPurge API (#8068).
+- Remove `agent_factory_watcher_sleep_time_in_seconds` from server_config.json (#8069).
+- Remove `igetwild` (#8084).
+- Remove `ilocate` (#8084).
+- Remove GSI authentication objects (#8257).
+- Remove Kerberos authentication objects (#8257).
+- Remove `--wlock` and `--rlock` options (#8273).
+- Remove osauth and `genOSAuth` (#8300).
+- Remove ZeroMQ references from buildsystem (#8326).
+- Remove Avro references from buildsystem (#8327).
+- Remove `rodsDebug` member variable from `RodsEnvironment` struct (#8367).
+- Remove control plane member variables from `RodsEnvironment` struct (#8368).
+- Remove option to read server_config.json from `new_database_connection` function (#8438, #8445).
+- Remove support for "irodsOdbcDSN" environment variable (#8445).
+- Remove unnecessary iadmin subcommands - `ctime`, `mkdir`, `rmdir`, `ls` (#8497).
+- Remove code which prints/logs network messages between client and server (#8509).
+
+### Deprecated
+
+- Deprecate legacy authentication plugins (#8257).
+- Deprecate `clientLogin`-related functions (#8257).
+
+### Fixed
+
+- Replace ad-hoc GenQuery1 parser with flex-bison parser (#3902, #4983, #5992, #7164, #7169, #7174, #7302, #7972, #8033).
+- Reduce scope of `FILE_PATH_KW` (#4932).
+- Prevent `iinit` from showing native authentication password prompt for non-native authentication schemes (#7948).
+- Fix error handling logic for heartbeat operation (#8050).
+- Make GenQuery2 honor group permissions (#8259).
+- Restore support for `KeyValPair` in Python rule engine plugin (#8265).
+- Fix redirect bug of client request from Consumer server to Provider server in GenQuery2 API (#8439).
+- Unregister replica before unlinking in DataObjUnlink API (#8441).
+- Set working directory when stopping server during package upgrade (#8492).
+- Replace invalid UTF-8 sequences in log messages (#8498).
+
+### Added
+
+- Add dedicated server_config.json property for specifying the server's identity (#7229).
```

Comment:
Needs to state the property - `"host"`. As it is, no one knows what this refers to.

Reply:
Done.